charts/kubewarden-controller/templates/cert-tls.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.tls.useCertManager .Values.tls.useSelfSignedCert }}
```

Comment:
None of these values are defined inside of the `values.yaml` file. I think it would be better to have them added

Reply:
They are there, see https://github.com/kubewarden/helm-charts/blob/f8f6ebf05da963f1d5670a5e853fcc6e2c1c21ef/charts/kubewarden-controller/values.yaml#L36-L38.

Reply:
Sorry... let's add some comments explaining their purpose.